modopt/opt/cost.py

```diff
@@ -183,7 +185,9 @@ def get_cost(self, *args, **kwargs):
         """
 
         # Check if the cost should be calculated
-        if self._iteration % self._cost_interval:
+        if self._cost_interval is None:
+            test_result = False
+        elif self._iteration % self._cost_interval:
```

Comment:
Isn't this more of an `or` than an `elif` condition?

Reply:
Yes, just thought of enunciating it, but well changed it to `or`